tests/unit/features/test_parts.py

```diff
@@ -0,0 +1,203 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2023 Canonical Ltd.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 3 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from textwrap import dedent
+
+import pytest
+
+from craft_parts import parts
+from craft_parts.parts import Part
+
+
+@pytest.mark.usefixtures("enable_all_features")
+class TestPartPartitionUsage:
```

Comment:
are these the same tests as in `unit/features/partitions/test_parts.py`?

Reply:
Similar except:
1. `overlay` feature enabled
2.  partitions are used and tested in the `overlay` fileset in parts